### PR TITLE
chore: minor code refactoring

### PR DESCRIPTION
**Pull request by jamesgeorge007**
**What kind of change does this PR introduce?**
refactoring

**Did you add tests for your changes?**
Nope

**If relevant, did you update the documentation?**
N/A

**Summary**
Make use of object destructuring assignment approach and a couple of other minor tweaks.

**Does this PR introduce a breaking change?**
Nope

**Other information**
N/A

### CODE REVIEW

File: lib/utils/commands.js [hunk 1]
```diff
@@ -1,12 +1,11 @@
-const arguments = require('./cli-flags');
+const { commands } = require('./cli-flags');
 
-const info = arguments.commands;
-const names = Object.keys(info).map( i => {
-    if (info[i].alias) {
-        return [info[i].name, info[i].alias];
+const names = Object.keys(commands).map( i => {
```
Comment (ematipico) [inline lib/utils/commands.js]: `i` can be called `key`

File: lib/webpack-cli.js [hunk 2]
```diff
@@ -11,6 +11,11 @@ class WebpackCLI extends GroupHelper {
         this.groupMap = new Map();
         this.groups = [];
         this.processingErrors = [];
+        this.commandNames = {
+            'create': 'init',
```
Comment (ematipico) [inline lib/webpack-cli.js]: You can remove the apostrophe from the key, not needed

File: lib/webpack-cli.js [hunk 3]
```diff
@@ -11,6 +11,11 @@ class WebpackCLI extends GroupHelper {
         this.groupMap = new Map();
         this.groups = [];
         this.processingErrors = [];
+        this.commandNames = {
```
Comment (ematipico) [inline lib/webpack-cli.js]: Main this could be called `defaultCommands`. What do you think?

File: lib/webpack-cli.js [hunk 4]
```diff
@@ -152,19 +157,12 @@ class WebpackCLI extends GroupHelper {
...
-        } else if (command.name === 'plugin') {
-            command.name = 'generate-plugin';
-        }
-        return await require('./commands/external').run(command.name, ...args);
+        return await require('./commands/external').run(this.commandNames[command.name], ...args);
     }
 
     runHelp(args) {
         const HelpGroup = require('./groups/help');
-        const names = require('./utils/commands').names;
```
Comment (ematipico) [inline lib/webpack-cli.js]: What's the reason of this change?

Comment (jamesgeorge007) [reply lib/webpack-cli.js]: Object destructuring assignment makes the respective code snippet much concise.

File: lib/webpack-cli.js (hunk 3 above)
Comment (jamesgeorge007) [reply lib/webpack-cli.js]: yeah, makes sense :+1:

File: lib/utils/commands.js [hunk 5]
```diff
@@ -1,12 +1,11 @@
-const arguments = require('./cli-flags');
+const { commands } = require('./cli-flags');
 
-const info = arguments.commands;
-const names = Object.keys(info).map( i => {
-    if (info[i].alias) {
-        return [info[i].name, info[i].alias];
+const names = Object.keys(commands).map( key => {
+    if (commands[key].alias) {
+        return [commands[key].name, commands[key].alias];
```
Comment (evenstensberg) [inline lib/utils/commands.js]: I think you could use destruct syntax here?

File: lib/webpack-cli.js [hunk 6]
```diff
@@ -11,6 +11,11 @@ class WebpackCLI extends GroupHelper {
         this.groupMap = new Map();
         this.groups = [];
         this.processingErrors = [];
+        this.defaultCommands = {
```
Comment (evenstensberg) [inline lib/webpack-cli.js]: Nice. You can move this outside the class

**Comment by ematipico** [comment]
Thank you! 
FYI, this is not a feature (`feat`). That's why I changed to a simple chore
